samples/nrf_desktop/src/modules/usb_state.c

```diff
@@ -28,7 +29,30 @@ static enum usb_state state;
 static u8_t hid_protocol = HID_PROTOCOL_REPORT;
 static struct device *usb_dev;
 
-static atomic_t forward_status;
+static struct config_channel_frame frame;
+static atomic_t status;
+static struct k_delayed_work timeout;
```

Comment:
I think we should have this in common place for both ble and usb. Reason is not only to avoid duplication.
If we start transactions on both USB and BLE, there seem to be no way of distinguishing which data to direct where and so we may have a problem with formulating a correct reply.

Reply:
Noted. We need to remember from which transport (and connection) the request came + handle disconnects of BLE/USB